README.md

```diff
@@ -19,3 +19,14 @@ quantitatively reason with them.
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+We should start with the definition of a good pivot. A good pivot creates two partitions of size at most 3n/4. That means inside of a sorted list a good pivots exists between n/4 and 3n/4. This means that half of the elements are good pivots in a random list. This means the possibility of picking a good pivot by selecting the first, last, or random elements is all the same at 50%. 
+The Median-of-three method considers this when attempting to pick a good pivot. It takes the medium of the first, middle, and last elements. Each element has a 50% chance of being a good pivot. It also knows that inside a sorted list, good pivots exist between n/4 and 3n/4, so it takes the medium of the three elements. This means the Median-of-three method has an 87.5% chance of picking any good pivot.
+I got that by taking the percentage chance of each of the elements not being a good pivot, which is 50%, and multiplying them together
+.5 *.5*.5 =.125, Meaning there is a 12.5% chance of the Median-of-three method not finding a good pivot and then 1 - .125 to get the percent chance of at least one being a good pivot 87.5%. Selecting the median of the three elements is the best way to figure out which is a good pivot. So the Median-of-three percent chance of picking a good pivot is 87.5%.
```

Comment:
But here you're not taking into account that the median is taken -- not all three elements need to be good pivots for the median to be a good pivot.